# File describing possible positions of the arm
# Safe Limits
# grip: [0..1]
# base: [1..2]
# base_to_bone: [0..-2]
# bone_to_hand: [-2..2]
# hand_to_grip: [-1.57..1.57]

initial_pos:
    base: 0
    base_to_bone: 2
    bone_to_hand: 2
    hand_to_grip: 0
close_grip:
    grip: 1.3
release_grip:
    grip: 0
extended_hand:
    base: 1.57
    base_to_bone: 0
    bone_to_hand: 0
    hand_to_grip: 0
to_grasp:
    base: 1.57
    base_to_bone: 1
    bone_to_hand: -1.57
    hand_to_grip: 0
unknock:
    base: 1.81
knock:
    base: 1.65
side:
    base: 1.28
    base_to_bone: 0
    bone_to_hand: -0.41
    hand_to_grip: 0
far:
    base: 1.0
    base_to_bone: 0
    bone_to_hand: -0.41
    hand_to_grip: 0
